blackbox/docs/configuration.txt

```diff
@@ -1868,6 +1868,52 @@ The meaning of the fields of the are as follows:
         method: trust
         protocol: pg
 
+.. _ssl_config:
+
+Secured Communications - SSL/TLS
+--------------------------------
+
+    Secured communications via SSL allows you to encrypt traffic between CrateDB nodes and clients
+    connecting to them. Connections are secured using Transport Layer Security (TLS).
+
+**ssl.psql.enabled**
+  | *Runtime:*  ``no``
+  | *Default:* ``false``
+
+  Set this to true to enable secured communications between CrateDB node and client via SSL
+  via PostgreSQL wire protocol.
+
+**ssl.keystore_filepath**
+  | *Runtime:* ``no``
+
+  The full path to the node keystore file.
+
+**ssl.keystore_password**
+  | *Runtime:* ``no``
+
+  The password used to decrypt the keystore file defined with ```ssl.keystore_filepath``.
+
+**ssl.keystore_key_password**
+  | *Runtime:* ``no``
+
+  The password entered at the end of the keytool -genkey command if different than the ``ssl.keystore_password``.
```

Comment:
The key password has to be set explicitly. It doesn't use the keystore password if none is set.